classes/Kohana/Jam/Range.php

```diff
@@ -59,13 +59,13 @@ public function __construct($source = NULL, $format = NULL)
 	{
 		if (is_string($source) AND $source !== '')
 		{
-			$source = explode('|', $source);
+			$source = array_map('intval', explode('|', $source));
 		}
 
 		if (is_array($source))
 		{
-			$this->min($source[0]);
-			$this->max($source[1]);
+			$this->min((int) $source[0]);
```

Comment:
@EZarkov You can add the `array_map` call after converting the string to an array, this way will be more consistent.